docs/getting-started.md

```diff
@@ -15,7 +15,7 @@ export DOCKER_TAG=mybuild
 Then build the manifests and images:
 
 ```bash
-make && make push
+make && make push && make manifests
```

Comment:
running `make` is equivalent to running `make all`. The make all command running the `./hack/build-manifiests.sh`command the same as the `make all` command. It looks like the difference is that in `make all` we do not set the following env vars. 

```
CSV_VERSION=${CSV_VERSION} 
QUAY_REPOSITORY=${QUAY_REPOSITORY}
PACKAGE_NAME=${PACKAGE_NAME} 
KUBEVIRT_INSTALLED_NAMESPACE=${KUBEVIRT_INSTALLED_NAMESPACE} 
``` 
				
Should we instead update the `make all` command to include these env vars?

Reply:
Not sure I understand how those env vars can help. If they are added to `make all` then still they should be passed from the outside (e.g. like `DOCKER_PREFIX`):
```bash
export DOCKER_PREFIX=registry:5000
export CSV_VERSION=???
make all
```
Also I dont see how they impact the SHA sums of the images. Or maybe I misunderstood the suggestion?

`make all` indeed includes the building of the manifests. The problem is that it does not include the `push` target which in turn produces the new SHA sums in `bazel-bin/push-<image-name>.digest` 

https://github.com/kubevirt/kubevirt/blob/7eed0d27e61297f803e9e144d2ef20d789e35110/hack/parse-shasums.sh#L29-L33

Those are consumed by the `build-manifests.sh`:

https://github.com/kubevirt/kubevirt/blob/7eed0d27e61297f803e9e144d2ef20d789e35110/hack/build-manifests.sh#L57-L58

https://github.com/kubevirt/kubevirt/blob/7eed0d27e61297f803e9e144d2ef20d789e35110/hack/build-manifests.sh#L94-L98

Therefore if the manifests are not regenerated after `make push` they will contain the old SHA sums.

Reply:
Can it be that it all boils down to the fact that the order in `make all` is  wrong? I think they should be created after `bazel-build`. Bazel will either use the cache or build and after that we have the digests.

Reply:
Well, from what I can tell the new digests are generated only after pushing the image. `make all` does not seem to include the push target. Though I dont know, maybe smth has changed as I havent checked that for a while. Since I faced that problem I have been always doing `make manifests` at the end.

Reply:
You are absolutely right. Approved.

It may make sense to re-evaluate when we call the manifest creator. We should probably always call it together with the push command.